docs/src/site/optics.md

```diff
@@ -6,5 +6,12 @@ section: "optics"
 
 # Optics
 
+## Convert between two types
```

Comment:
I am not sure we should make that kind of categorisation here. Maybe it would be a good place to put the class diagram that shows relationship between optics.